src/panels/lovelace/cards/hui-entity-button-card.ts

```diff
@@ -103,7 +107,13 @@ class HuiEntityButtonCard extends hassLocalizeLitMixin(LitElement)
     `;
   }
 
-  private renderStyle() {
+  protected updated(_changedProperties: PropertyValues): void {
+    if (this.hass && this._config) {
```

Comment:
thinking now, we only have to apply a theme if:

 - this was the first render
 - the theme has changed